packages/paypal-commerce-integration/src/PaypalCommerceRatePayPaymentMethod.tsx

```diff
@@ -9,6 +9,11 @@ import { DynamicFormField, DynamicFormFieldType, FormContext } from '@bigcommerc
 import { FormField } from '@bigcommerce/checkout-sdk';
 import getPaypalCommerceRatePayValidationSchema from './validation-schemas/getPaypalCommerceRatePayValidationSchema';
 import { LoadingSpinner } from '@bigcommerce/checkout/ui';
+import { CustomError } from '../../payment-integration-api/src/errors';
```

Comment:
This doesn't seem to be the right way of importing, we should be doing from namespace like previous line.